client/my-sites/upgrades/domain-search/domain-search.jsx

```diff
@@ -158,25 +148,19 @@ var DomainSearch = React.createClass( {
 			<Main className={ classes }>
 				<SidebarNavigation />
 				{ content }
+				<QueryProductsList />
+				<QuerySitePlans siteId={ selectedSite.ID } />
 			</Main>
 		);
 	}
-} );
-
-module.exports = connect(
-	function( state, props ) {
-		return {
-			sitePlans: getPlansBySite( state, props.sites.getSelectedSite() ),
-			domainsWithPlansOnly: currentUserHasFlag( state, DOMAINS_WITH_PLANS_ONLY )
-		};
-	},
-	function( dispatch ) {
-		return {
-			fetchSitePlans( sitePlans, site ) {
-				if ( shouldFetchSitePlans( sitePlans, site ) ) {
-					dispatch( fetchSitePlans( site.ID ) );
-				}
-			}
-		};
-	}
-)( DomainSearch );
+}
+
+export default connect(
+	( state ) => ( {
+		selectedSite: getSelectedSite( state ),
```

Comment:
Since we're calling `getSelectedSite()` more than once here, perhaps we should be storing it in a `const`?